pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -91,7 +91,7 @@ func status() registry.State {
 	ctx, cancel := context.WithTimeout(context.Background(), 8*time.Second)
 	defer cancel()
 
-	cmd := exec.CommandContext(ctx, path, "-NoProfile", "-NonInteractive", "@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent")
+	cmd := exec.CommandContext(ctx, path, "-NoProfile", "-NonInteractive", "@(Get-CimInstance Win32_ComputerSystem).HypervisorPresent")
```

Comment:
will this work on Old windows versions ?

how about we put the old way  as Fail Over option ? if this fails lets try the old way (maybe someone with old windows) ?

Reply:
I think that will be a nice addition
Should add it and you can review the changes for improvement ?